### PR TITLE
Added the Weaviate vector search podcast

### PR DESCRIPTION
**Pull request by bobvanluijt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,7 @@ This list started from a discussion in DataTalks.Club. Want to talk about data?
 
 ## Data Enginering, Big Data, etc
 
+- [Weaviate vector search podcast](https://weaviate.io/podcast.html)
```
Comment (alexeygrigorev) [inline README.md]: Can you please add it to the end of the list?

Comment (alexeygrigorev) [reply README.md]: I mean after the airflow pod

Comment (bobvanluijt) [reply README.md]: Ok - it's now _after_ Airflow. Please let me know if this is ok.

**Comment by alexeygrigorev** [comment]
Thanks

**Comment by bobvanluijt** [comment]
> Thanks

no, no, thank you 😉👍

**Pull request merged**